pom.xml

```diff
@@ -142,11 +142,29 @@
             <version>3.14.1</version>
         </dependency>
 
-        <!-- PUT BACK AS TEST -->
+        <dependency>
+            <groupId>org.apache.tuweni</groupId>
+            <artifactId>tuweni-eth</artifactId>
+            <version>1.1.0-SNAPSHOT</version>
```

Comment:
is there a released version? we won't be able to release with a SNAPSHOT version

Reply:
I think we should get a release tomorrow but I will check with Antoine. If it can wait until tomorrow or a little bit more then we can take their release. Otherwise we can always do our own for tuweni